src/coreclr/jit/optcse.cpp

```diff
@@ -2371,13 +2398,6 @@ void CSE_HeuristicParameterized::CaptureLocalWeights()
             continue;
         }
 
-        // Only consider for integral types
-        //
-        if (varTypeIsFloating(varDsc->TypeGet()) || varTypeIsMask(varDsc->TypeGet()))
-        {
-            continue;
-        }
```

Comment:
@AndyAyersMS should this be changed?

Reply:
I wasn't sure if this code was still "live" as many of the other scripts, like the python one, it mentioned aren't in the repo at all.

Reply:
Talked with @AndyAyersMS offline. He suggested leaving the code untouched and adding a comment indicating that it should be fixed the next time that code is touched/looked at.

-- If the scripts are available, I'd be happy to do that in a follow up PR, just none of the `./src/coreclr/scripts/cse_ml/*` files appear to exist